src/components/NavBar.jsx

```diff
@@ -1,66 +1,104 @@
-import React, {useState, useEffect} from 'react'
-import { Link } from 'react-router-dom';
-import { Button } from './Button';
-import './navbar.css';
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { Button } from "./Button";
+import "./navbar.css";
 
 function NavBar() {
-    const [click, setClick] = useState(false);
-    const [button, setButton] = useState(true);
+  const [click, setClick] = useState(false);
+  const [button, setButton] = useState(true);
 
-    const handleClick = () => setClick(!click);
-    const closeMobileMenu = () => setClick(false);
+  const handleClick = () => setClick(!click);
+  const closeMobileMenu = () => setClick(false);
 
-    const showButton = () => {
-        if(window.innerWidth <= 960) {
-            setButton(false);
-        } else {
-            setButton(true);
-        }
-    };
+  const showButton = () => {
```

Comment:
This should've been handled using css only, you can use [media-queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) to hide and show elements.